agent/taskresource/logrouter/logrouter_unimplemented.go

```diff
@@ -0,0 +1,135 @@
+func (c *LogRouterResource) DesiredTerminal() bool {
+	return false
+}
+
+// KnownCreated returns true if the resource's known status is CREATED.
+func (c *LogRouterResource) KnownCreated() bool {
+	return false
+}
+
+// TerminalStatus returns the last transition state of log router resource.
+func (c *LogRouterResource) TerminalStatus() resourcestatus.ResourceStatus {
+	return resourcestatus.ResourceStatusNone
+}
+
+// NextKnownState returns the state that the resource should progress to based
+// on its `KnownState`.
+func (c *LogRouterResource) NextKnownState() resourcestatus.ResourceStatus {
+	return resourcestatus.ResourceStatusNone
+}
+
+// ApplyTransition calls the function required to move to the specified status.
+func (c *LogRouterResource) ApplyTransition(nextState resourcestatus.ResourceStatus) error {
+	return errors.New("not implemented")
+}
+
+// SteadyState returns the transition state of the resource defined as "ready".
+func (c *LogRouterResource) SteadyState() resourcestatus.ResourceStatus {
+	return resourcestatus.ResourceStatusNone
+}
+
+// SetKnownStatus safely sets the currently known status of the resource.
+func (c *LogRouterResource) SetKnownStatus(status resourcestatus.ResourceStatus) {}
+
+// SetAppliedStatus sets the applied status of resource and returns whether
+// the resource is already in a transition.
+func (c *LogRouterResource) SetAppliedStatus(status resourcestatus.ResourceStatus) bool {
+	return false
+}
+
+// GetKnownStatus safely returns the currently known status of the resource.
+func (c *LogRouterResource) GetKnownStatus() resourcestatus.ResourceStatus {
+	return resourcestatus.ResourceStatusNone
+}
+
+// SetCreatedAt sets the timestamp for resource's creation time.
+func (c *LogRouterResource) SetCreatedAt(createdAt time.Time) {}
+
+// GetCreatedAt sets the timestamp for resource's creation time.
+func (c *LogRouterResource) GetCreatedAt() time.Time {
+	return time.Time{}
+}
+
+// Create creates the log router resource.
+func (c *LogRouterResource) Create() error {
+	return errors.New("not implemented")
+}
+
+// Cleanup cleans up the log router resource.
+func (c *LogRouterResource) Cleanup() error {
+	return errors.New("not implemented")
+}
+
+// StatusString returns the string of the log router resource status.
+func (c *LogRouterResource) StatusString(status resourcestatus.ResourceStatus) string {
+	return "undefined"
+}
+
+// GetTerminalReason returns the terminal reason for the resource.
+func (c *LogRouterResource) GetTerminalReason() string {
+	return "undefined"
+}
+
+// MarshalJSON marshals LogRouterResource object.
+func (c *LogRouterResource) MarshalJSON() ([]byte, error) {
+	return nil, nil
+}
+
+// UnmarshalJSON unmarshals LogRouterResource object.
+func (c *LogRouterResource) UnmarshalJSON(b []byte) error {
+	return nil
+}
+
+// Initialize fills in the resource fields.
+func (cgroup *LogRouterResource) Initialize(resourceFields *taskresource.ResourceFields,
```

Comment:
nit: Calling the LogRouterResource a `cgroup` or `c` looks like code copy pasta :P

Reply:
good catch. fixed